src/ios/SOSPicker.m

```diff
@@ -236,16 +244,20 @@ - (void)assetsPickerController:(GMImagePickerController *)picker didFinishPickin
                 break;
             } else {
                 if(self.outputType == BASE64_STRING){
-                    [result_all addObject:[data base64EncodedStringWithOptions:0]];
+                    [resultList addObject:[data base64EncodedStringWithOptions:0]];
                 } else {
-                    [result_all addObject:[[NSURL fileURLWithPath:filePath] absoluteString]];
+                    [resultList addObject:[[NSURL fileURLWithPath:filePath] absoluteString]];
                 }
             }
         }
     }
 
     if (result == nil) {
-        result = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsArray:result_all];
+        for(int i = 0; i < resultList.count; i++) {
+            NSDictionary *imageInfo = @{@"path": resultList[i], @"width": imageSize[i][@"width"], @"height": imageSize[i][@"height"]};
```

Comment:
is there a way where we can feed the array of object { path: xxx, width, xx, height: xxx } at one shot ? instead of playing with indexes in the different arrays ?